### ISSUE TITLE
training gc_pgp model

### ISSUE THREAD

**Issue by ynw2021** [autonomousvision/tuplan_garage#20]
Hi,

For you released gc_pgp code,I notice you pretrained the aggregator. Did you load the pretrained aggregator when training the gc_pgp? I didn't find any model loading code. Also,could you share the training configuration of urban driver? I trained the urbandriver on the train150k set with 4000 scenarios per type, but the performance is much lower than it in the table in your readme, with only 55 open-loop score.

Bests,

**Comment by DanielDauner** [comment]
Thanks for your interest in our project! 

Indeed, the GC-PGP training script was missing to load the model weights from pretraining. It should be now fixed in [`train_gc_pgp.sh`](https://github.com/autonomousvision/tuplan_garage/blob/650d3e68deb3d1bba84ab3d0a53a3ac394866db1/scripts/training/train_gc_pgp.sh#L59C2-L59C2) by adding the parameter `lightning.trainer.checkpoint.resume_training=true`. Thanks for pointing this out! 

You can find the training script for Urban Driver here: [`train_urban_driver.sh`](https://github.com/autonomousvision/tuplan_garage/blob/650d3e68deb3d1bba84ab3d0a53a3ac394866db1/scripts/training/train_urban_driver.sh). Potentially, the training objective is essential here. We used an L1-Loss. 

I hope that helps!

**Comment by ynw2021** [comment]
Thank you so much for your reply！When I train the gc_pgp model，I find this warning：

【2023-10-25 20:08:15】2023-10-25 20:08:09,363 INFO {/nfs/volume-904-5/gwynwang/code/nuplan-devkit-master/nuplan/planning/script/builders/utils/utils_config.py:199} Updating torch.optim.lr_scheduler.MultiStepLR in ddp setting is not yet supported. Learning rate scheduler config will not be updated.
【2023-10-25 20:08:15】2023-10-25 20:08:09,363 INFO {/nfs/volume-904-5/gwynwang/code/nuplan-devkit-master/nuplan/planning/script/builders/utils/utils_config.py:245} Optimizer and LR Scheduler configs updated according to ddp strategy.

which might mean that the learning rate scheduler is not working under the default ddp accelerator. 
When I check the tensorboard, I find the initial learning rate set as 2e-4, not 1e-4, and not decaying.
![image](https://github.com/autonomousvision/tuplan_garage/assets/49264913/2d08f180-1752-4930-8e10-8b493847fb87)
Could you help me with this?

**Comment by DanielDauner** [comment]
Are you using multiple GPUs? The nuplan-devkit scales the learning-rate times the number of GPUs, which can can be disabled in this config:  [`default_lightning.yaml`](https://github.com/motional/nuplan-devkit/blob/master/nuplan/planning/script/config/training/lightning/default_lightning.yaml#L2C17-L2C18). 

Unfortunately, I have little experience training our models on multiple GPUs. We always trained with a single GPU.

**Comment by ynw2021** [comment]
Thanks for your reply, it is very helpful!
